x-pack/plugins/synthetics/public/apps/synthetics/synthetics_app.tsx

```diff
@@ -98,30 +99,32 @@ const Application = (props: SyntheticsAppProps) => {
                 fleet: startPlugins.fleet,
               }}
             >
-              <Router history={appMountParameters.history}>
-                <EuiThemeProvider darkMode={darkMode}>
-                  <SyntheticsRefreshContextProvider>
-                    <SyntheticsSettingsContextProvider {...props}>
-                      <SyntheticsThemeContextProvider darkMode={darkMode}>
-                        <SyntheticsStartupPluginsContextProvider {...startPlugins}>
-                          <div className={APP_WRAPPER_CLASS} data-test-subj="syntheticsApp">
-                            <RedirectAppLinks
-                              className={APP_WRAPPER_CLASS}
-                              application={core.application}
-                            >
-                              <InspectorContextProvider>
-                                <PageRouter />
-                                <ActionMenu appMountParameters={appMountParameters} />
-                                <TestNowModeFlyoutContainer />
-                              </InspectorContextProvider>
-                            </RedirectAppLinks>
-                          </div>
-                        </SyntheticsStartupPluginsContextProvider>
-                      </SyntheticsThemeContextProvider>
-                    </SyntheticsSettingsContextProvider>
-                  </SyntheticsRefreshContextProvider>
-                </EuiThemeProvider>
-              </Router>
+              <ObservabilityAIAssistantProvider value={startPlugins.observabilityAIAssistant}>
```

Comment:
Can you please add some screenshot or link to the issue? which feature is being added here and some instructions to test the changes?

Reply:
> Adds the action menu item (that opens a flyout) everywhere in Observability apps.

😛

Reply:
let me ask ChatGpt what's  Observability AI Assistant

Reply:
@shahzad31 mind approving?